ckanext/canada/i18n/fr/LC_MESSAGES/canada.po

```diff
@@ -1760,6 +1760,12 @@ msgstr "En attente"
 msgid "Published"
 msgstr "Publié"
 
+msgid "title_string asc"
+msgstr "title_fr asc"
+
+msgid "title_string desc"
+msgstr "title_fr desc"
```

Comment:
it feels pretty strange to use gettext to translate a solr search parameter, maybe just use a if-else in the sort_by snippet instead.